trieste/models/gpflux/architectures.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""
+This file contains wrappers for some implementations of basic GPflux architectures.
+"""
+
+from __future__ import annotations
+
+import numpy as np
+import tensorflow as tf
+from gpflux.architectures import Config, build_constant_input_dim_deep_gp
+from gpflux.models import DeepGP
+
+from ...types import TensorType
+
+tf.keras.backend.set_floatx("float64")
```

Comment:
this is important bit but so far the practice was to leave setting floaat type globally to a user, hence we should put it in the tests and notebooks but not here in trieste code

Reply:
Ok, I have removed this and will add a note in docstrings somewhere so the user is aware as well.